### PR TITLE
Add warning for lammps pre-processing

### PR DESCRIPTION
**Pull request by timcallow**
Warning for users when pre-processing SNAP with lammps.  This is a temporary workaround for issue #502

### CODE REVIEW

File: mala/descriptors/descriptor.py [hunk 1]
```diff
@@ -526,6 +526,9 @@ def _setup_lammps(self, nx, ny, nz, outdir, lammps_dict,
         """
         from lammps import lammps
 
+        printout("Warning: do not initialize more than one pre-processing calculation\
+in the same directory at the same time. Data may be over-written.")
```
Comment (RandomDefaultUser) [inline mala/descriptors/descriptor.py:530]:
```suggestion
parallel_warn("Do not initialize more than one pre-processing calculation\
in the same directory at the same time. Data may be over-written.")
```

**Review by RandomDefaultUser — CHANGES_REQUESTED** [review]
Thanks for the PR @timcallow, I agree that for now (until we have fixed the underlying issue) a warning like this is the best way to proceed. MALA has a parallel warning function that I would suggest we use instead of the printout function.

**Comment by timcallow** [comment]
Thanks for the suggestion, I didn't notice that function - I've committed your change

**Review by RandomDefaultUser — APPROVED** [review]
Thanks, looks good to me now! :)

**Pull request merged**